### DISCUSSION TITLE
Does it work with OpenTK.GLWpfControl ?

### DISCUSSION THREAD

**Discussion by Pathompongmc** [hjoykim/THREE#21] [General]
Does it work with OpenTK.GLWpfControl ? 
Because I don't need to use WindowsFormsHost , it make render screen always top most on my wpf application.

**Comment by hjoykim** [comment]
Hello,
To exclude winformhost, I applied GLWpfGLControl before, but
There were some errors in running three.js shader functions using OpenTK.GLWpfGLControl
but I couldn't bear to debug GLWpfGLControl.
I'm still trying to do it in a different way.
If you know of any related errors, please share.

**Comment by Pathompongmc** [comment]
Can be Fixed by AirspacePopup. 😀
https://stackoverflow.com/questions/6087835/can-i-overlay-a-wpf-window-on-top-of-another/6452940#6452940

**Comment by hjoykim** [comment]
I've commit sources using GLWpfControls
please refer to #53
